### ISSUE TITLE
Mark-up of event edit page

### ISSUE THREAD

**Issue by stephenharris** [stephenharris/Event-Organiser#280]
The mark-up here is a table, but this is solely for aesthetics - which is wrong. Tabular data is not being presented (a11y issue), and we could get a much better 'responsive' mark-up.

Other issues include:
 - fields without labels
 - perhaps superfluous text
 - event summary should be a live region to prompt the user of the current state of the event after each change.
 - field IDs do not confirm to any pattern.

**Comment by stephenharris** [comment]
These issues have now been largely fixed. A bugs found, please open a new ticket.
